src/trackedit/internal/au3/au3trackeditclipboard.cpp

```diff
@@ -3,50 +3,35 @@
 */
 
 #include "au3trackeditclipboard.h"
+
+#include "Track.h"
+#include "containers.h"
 #include "WaveClip.h"
 #include "WaveTrack.h"
```

Comment:
please follow our codestyle about includes https://github.com/musescore/MuseScore/wiki/CodeGuidelines#header-and-implementation